src/index.html

```diff
@@ -7,6 +7,325 @@
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home" class="header__logo-link">
+        <img src="images/logo.png" alt="logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <a
+            href="#apple"
+            class="is-active"
+          >
+            <li>Apple</li>
+          </a>
```

Comment:
```suggestion
        <ul class="nav__list">
          <li>
            <a
              href="#apple"
              class="is-active"
            >
              Apple
            </a>
          </li>
```

Reply:
but in such case only txt will be link and task was that whole button (above and below) must be link